examples/custom_logger.js

```diff
@@ -0,0 +1,60 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+const Pulsar = require('../index.js');
+
+(async () => {
+    // Create a client
+    const client = new Pulsar.Client({
+        serviceUrl: 'pulsar://localhost:6650',
+        operationTimeoutSeconds: 30,
+    });
+
+    Pulsar.Client.setLogHandler((level, file, line, message) => {
+        console.log('[%s][%s:%d] %s', Pulsar.LogLevel.toString(level), file, line, message);
```

Comment:
Actually the `Pulsar.LogLevel.toString(level)` doesn't work.

Reply:
Could you share your log? It works fine on my local env.

Reply:
![image](https://user-images.githubusercontent.com/18204803/205266796-ae6562a7-b824-4e5b-bd4f-aa3314927b1f.png)

Reply:
@BewareMyPower Did you completely pull the PR? Looks like the `index.js` haven't been changed.

Reply:
Seems `require('Pulsar.node')` and `require('index.js')` has a different effect. I wonder if it's a problem.

Reply:
I see. I just copied your `custom_logger.js` to cover the `producer.js`. It works well after applying the change of `index.js`.